contracts/0.4.24/nos/NodeOperatorsRegistry.sol

```diff
@@ -475,6 +569,8 @@ contract NodeOperatorsRegistry is AragonApp, Versioned {
         _saveOperatorTargetValidatorsStats(_nodeOperatorId, operatorTargetStats);
 
         emit TargetValidatorsCountChanged(_nodeOperatorId, _targetLimit);
+
+        _updateTotalMaxValidatorsCount(_nodeOperatorId);
```

Comment:
After setting the target limit we potentially modify it again in the `_updateTotalMaxValidatorsCount()`?

Reply:
yep, but the expectation of `updateTargetValidatorsLimits` method call frequency is quite low, so we can neglect the optimization of reading the storage, which is also already "warmed up", in favor of reducing the bytecode